generator/vulkan/build_integration.zig

```diff
@@ -73,7 +67,29 @@ pub const GenerateStep = struct {
         const self = @fieldParentPtr(GenerateStep, "step", step);
         const cwd = std.fs.cwd();
 
+        var man = self.builder.cache.obtain();
+        defer man.deinit();
+
         const spec = try cwd.readFileAlloc(self.builder.allocator, self.spec_path, std.math.maxInt(usize));
+        // TODO: Look into whether this is the right way to be doing
+        // this - maybe the file-level caching API has some benefits I
+        // don't understand.
+        man.hash.addBytes(self.spec_path);
```

Comment:
Why is it required to add the spec's path here, too? Seems like that as long as the contents are the same, it wouldn't matter.